packages/@react-spectrum/list/src/styles.css

```diff
@@ -603,3 +603,82 @@
     box-shadow: inset 0 0 0 1px var(--spectrum-table-cell-border-color-key-focus);
   }
 }
+
+@media (forced-colors: active) {
+  .react-spectrum-ListView {
+    --spectrum-table-cell-border-color: CanvasText;
+    &:not(.react-spectrum-ListView--quiet) {
+      .react-spectrum-ListViewItem {
+        /* Box shadow for bottom border for non-selected rows that aren't immediately above a selected row (grey border bottom). */
+        /* Also omit bottom border for last row if the total content height of the ListView is equal or greater than the height of the container. This avoids overlapping bottom borders. */
+        &:not(.is-selected):not(.is-next-selected):not(.react-spectrum-ListViewItem--isFlushBottom) {
+          &:after {
+            forced-color-adjust: none;
+          }
+        }
+      }
+    }
+    .react-spectrum-ListViewItem {
+      &.is-selected {
+        /* Negative block start causes borders to actually be shared between items. Only works if we allow item overflow to be visible, like card focus rings. */
+        &:after {
+          --spectrum-global-color-blue-500: Highlight;
```

Comment:
don't override these, please make a new variable for this
the only thing we should be changing in here is `--spectrum-listview-*` prefixed variables

I'm not sure why there are also `--spectrum-selectlist` and `--spectrum-table` and `--spectrum-dropindicator` variables in here also being overridden, they should all be prefixed the same... that's not your fault, but it's something we need to address to make this right